pandas/core/reshape.py

```diff
@@ -974,6 +975,9 @@ def get_dummies(data, prefix=None, prefix_sep='_', dummy_na=False,
         Whether the dummy columns should be sparse or not.  Returns
         SparseDataFrame if `data` is a Series or if all columns are included.
         Otherwise returns a DataFrame with some SparseBlocks.
+    remove_first : bool, defalt False
```

Comment:
According to @TomAugspurger 's comment, I will change this option to `drop_first`.